tensorflow/core/util/device_name_utils.cc

```diff
@@ -162,6 +162,16 @@ bool DeviceNameUtils::ParseFullName(StringPiece fullname, ParsedName* p) {
       }
       progress = true;
     }
+    if (str_util::ConsumePrefix(&fullname, "/sycl:") ||
```

Comment:
I think in this particular case, since we're supporting SYCL in our official repo (optionally), having the kernel registrations / tests in the same files makes some sense.